tsconfig.json

```diff
@@ -8,7 +8,7 @@
     "sourceMap": true,
     "module": "esnext",
     "moduleResolution": "node",
-    "jsx": "react",
+    "jsx": "react-jsx",
```

Comment:
How did you come up with this? Is it necessary, that the compiler outputs this format for our tsx files?

Reply:
I didn't touch this file. Not sure what happened here.

Reply:
Perhaps I should revert this change and see what happens.